core/mm/core_mmu.c

```diff
@@ -968,6 +968,9 @@ static void collect_device_mem_ranges(struct tee_mmap_region *memory_map,
 	int subnode = 0;
 	int node = 0;
 
+	if (!fdt)
```

Comment:
I don't think this is supposed to happen. Wouldn't `assert(fdt);` make more sense?
 @LeiSen62

Reply:
The output of `get_manifest_dt()` is checked in `fip_sp_init_all()`

Reply:
I agree

Reply:
Ok for `assert()` then

Reply:
For commit "core: mmu: check pointer to manifest device tree": ok for assert()?
(it's the only commit that has no review tags)

Reply:
@clementfaure please add the `assert(fdt)` to "core: mmu: check pointer to manifest device tree" and check if Coverity is happy about it. Then you may add my `Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>` to that commit.

Reply:
Coverity is happy, I was waiting for a tag to force-push the PR.